integration_tests/features/WalletCli.feature

```diff
@@ -0,0 +1,115 @@
+Feature: Wallet CLI
+
+    Scenario: As a user I want to change base node for a wallet via command
```

Comment:
For clarity, I suggest adding "command line" instead of "command" in all instances

```suggestion
    Scenario: As a user I want to change base node for a wallet via command line
```